.github/workflows/app.yml

```
@@ -0,0 +1,45 @@
+name: Test and build Jellyfin Android
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Gradle wrapper and dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-java-${{ matrix.java }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-java-${{ matrix.java }}-gradle-
+            ${{ runner.os }}-java-
+      - name: Set environment variable with version
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "JELLYFIN_VERSION=${$GITHUB_REF | sed 's|refs/tags/||'}" >> $GITHUB_ENV
```

are you sure this syntax is correct?
```suggestion
        run: echo "JELLYFIN_VERSION=$(echo $GITHUB_REF | sed 's|refs/tags/||')" >> $GITHUB_ENV
```

However, an alternative that requires escaping though:
```suggestion
        run: echo "JELLYFIN_VERSION=${GITHUB_REF//refs\/tags\//}" >> $GITHUB_ENV
```

Something useful i love to use when it comes to bash scripts: [Bash cheatsheet](https://devhints.io/bash#parameter-expansions)

---

Now that you say it - no xD
I copied Niels' suggestion and didn't think about it. I like the alternative though. Gonna apply that one.

---

@nielsvanvelzen btw, could we also remove the 'v' prefix here instead of inside the `VersionUtils`?

---

See my other comment